packages/commercetools/theme/components/profile/PasswordResetForm.vue

```diff
@@ -0,0 +1,126 @@
+<template>
```

Comment:
**suggestion** In the project a convention is to name folders in `components` that group components specific to a certain page in a dir with a name of that page (eg `Checkout.vue` - page, `checkout` - dir in components) and other components either in a root or grouped if they're all about a specific feature. These components are part of `MyAccount` so lets rename that dir accordingly :)